npm/react/rollup.config.js

```diff
@@ -0,0 +1,79 @@
+import ts from 'rollup-plugin-typescript2'
+import resolve from '@rollup/plugin-node-resolve'
```

Comment:
Weren't these added in another PR? I wonder why these changes are showing up here.

Reply:
Edit: need to update the base branch but when I tried to do so, I got an error about a protected branch.